index.js

```diff
@@ -65,7 +75,7 @@ HtmlWebpackPlugin.prototype.apply = function (compiler) {
   compiler.plugin('emit', function (compilation, callback) {
     var applyPluginsAsyncWaterfall = self.applyPluginsAsyncWaterfall(compilation);
     // Get all chunks
-    var allChunks = compilation.getStats().toJson().chunks;
+    var allChunks = getStats(compilation).chunks;
```

Comment:
Yeah probably not :D

I was wrong here. So is everything still works with `getStats()` then without the need for `toJSON()`?